dags/sql/indicateurs_sip/schema.sql

```diff
@@ -0,0 +1 @@
+-- noinspection SqlNoDataSourceInspectionForFile
```

Comment:
le schema.sql est vide?

Reply:
fixed

Reply:
CREATE SCHEMA IF NOT EXISTS sinon peut-être ca peut planter si le schema existe déjà